packages/block-editor/src/hooks/background.js

```diff
@@ -367,6 +368,64 @@ function backgroundSizeHelpText( value ) {
 	return __( 'Set a fixed width.' );
 }
 
+const coordsToBackgroundPosition = ( value ) => {
```

Comment:
This is just a quick pass at the mapping — if this PR looks okay, I'll fine-tune the logic a bit here and add some tests.

Reply:
This particular bit LGTM, but just curious: why do we need to do this mapping? Aren't the percentage values enough?

Reply:
Good question: we need to have at least some form of mapping so that we're not storing values like `{ x: 0.5, y: 0.5 }` in the block attributes for `style.background.backgroundPosition`. For future proofing the feature, I think it's better if that value can be stored in the block delimiter as a real CSS value, rather than a value that's intended for the internal workings of the focal point picker. So at the very least, I think it's better if we can have the support map `{ x: 0.5, y: 0.5 }` to `50% 50%`.

In terms of mapping the natural language values like `left`, `center`, etc, that's mostly me thinking of what would be good to support for if/when we get to `theme.json` support where folks might be writing the JSON a bit by hand. It'd be nice to be able to support mapping potentially handwritten values so that they work naturally with this control.

For now, though, we only really need the generic decimal `0.5` > `50%` mapping for this to work, so I'm happy to pare it back for this PR.

Reply:
> In terms of mapping the natural language values like left, center, etc, that's mostly me thinking of what would be good to support for if/when we get to theme.json support where folks might be writing the JSON a bit by hand.

Very good idea!

Reply:
For now I've just gone with converting between percentage and coordinate values. There's a bit more nuanced to [the multi-value syntax](https://developer.mozilla.org/en-US/docs/Web/CSS/background-position) for background position, so I think that'd be better to handle separately in a follow-up code quality PR. For now I've gone with the main use case logic which is background position values that were created using this particular UI control.